<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="192.168.1.33" failures="1" tests="1" name="runners.TestRunner" time="7.435" errors="0" timestamp="2025-10-20T19:25:39 TRT" skipped="0">
  <testcase classname="runners.TestRunner" name="runScenario" time="7.435">
    <failure message="Test failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Test failed
at org.testng.Assert.fail(Assert.java:111)
at pages.BasePage.assertFail(BasePage.java:109)
at pages.CommonPage.navigateToMenu(CommonPage.java:40)
at stepdefinations.CommonSteps.theUserNavigatesToMenu(CommonSteps.java:13)
at âœ½.the user navigates to the following menu(file:///Users/katali/Desktop/TeknasyonAppiumQAFramework/src/test/resources/Feature/CustomTitleValidation.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runners.TestRunner -->
